packages/provider/src/api/authMiddleware.ts

```diff
@@ -29,18 +29,18 @@ export const authMiddleware = (env: ProviderEnvironment) => {
 
 			const { signature, timestamp } = extractHeaders(req);
 
-			if (!env.pair) {
+			if (!env.authAccount) {
```

Comment:
What about if a provider calls their admin API endpoint with their own pair, `env.pair`? This should also be allowed.

Reply:
Just to be clear - we want to attempt to verify with both the authAccount and the provider pair? For all routes?

Reply:
done